newIDE/app/src/MainFrame/index.js

```diff
@@ -811,6 +812,28 @@ const MainFrame = (props: Props) => {
     _onProjectItemModified();
   };
 
+  const onInstallExtension = (extensionShortHeader: ExtensionShortHeader) => {
+    //Close the extension tab before updating/reinstalling the extension
```

Comment:
Uber nitpicking: space after the //, and always a period at the end of comments.